### DISCUSSION TITLE
Integrazione Aqara Camera Hub G3 in HomeKit

### DISCUSSION THREAD

**Discussion by whirly69** [indomus/forum#3037] [HUB: Home Assistant]
Salve a tutti, ho da pochi gg installato Home Assistant Core tramite Docker su RPI 4B integrando senza difficolta l'hub philips hue, alcune prese Tuya attraverso l'account SmartThins, la serratura Nuki, l'apricancello Ring e appunto l'hub aqara in oggetto attraverso la procedura di disconnettere da home kit,  connettendolo a Home Assistant e poi ricollegandolo a Home kit attraverso l'integrazione HomeKit bridge con  un HASS bridge. A questo punto ho il problema della visualizzazione video in homekit che resta sempre in ricerca.... in giro ho letto e mi sembra di aver capito che bisogna modificare il tipo di stream che HA imposta sulla videocamera, ma non riesco a trovare l'impostazione. Il file configuration.yaml è composto da piche righe che richiamano script.yaml e automation.yaml, per cui credo che ci sia qualche altro modo per accedere alla configurazione....... sono pochi giorni che ho messo mano su Home Assistant e la foga di cercare informazioni mi ha fatto andare un po in confusione. Grazie a chiunque mi possa illuminare

**Comment by sdavides** [comment]
Ciao, si nota che sei all'inizio di HA, non si è capito nulla, tuttavia per l'aqara g3 ti invito a vedere:

https://github.com/sdavides/AqaraPOST-Homeassistant/

forse ho capito, l' integrazione HomeKit per il video supporta solo MJPEG infatti la mia G3 da media si vede a scatti.
Ho descritto come fare per il video su git. ciao

**Reply by whirly69** [reply]
Grazie sembra abbastanza chiaro, domani mi metto all'opera
